cmake/Modules/HandleCUDAComputeCapability.cmake

```diff
@@ -8,24 +8,44 @@
 # those without the CUDA_ prefix
 
 if (NOT CUDA_TARGET_COMPUTE_CAPABILITY)
-	if("$ENV{CUDA_SM}" STREQUAL "")
-		set(ENV{CUDA_INCLUDE_DIRS} "${CUDA_INCLUDE_DIRS}")
-		set(ENV{CUDA_CUDART_LIBRARY} "${CUDA_CUDART_LIBRARY}")
-		set(ENV{CMAKE_CXX_COMPILER} "${CMAKE_CXX_COMPILER}")
-		execute_process(COMMAND bash -c "${CMAKE_CURRENT_SOURCE_DIR}/scripts/get_cuda_sm.sh"  OUTPUT_VARIABLE CUDA_TARGET_COMPUTE_CAPABILITY_) 
+	if(WIN32)
+		find_package(CUDA)
```

Comment:
CUDA is now an integral part of CMake, we don't use find_package on it. All CUDA-related variables are built-in.